airbyte-webapp-e2e-tests/cypress/pages/replicationPage.ts

```diff
@@ -118,14 +124,56 @@ export const searchStream = (value: string) => {
   cy.get(streamNameInput).type(value);
 };
 
+export const clickSaveReplication = ({ reset = false, confirm = true } = {}) => {
+  cy.intercept("/api/v1/web_backend/connections/update").as("updateConnection");
+
+  submitButtonClick();
+
+  if (confirm) {
+    confirmStreamConfigurationChangedPopup({ reset });
+  }
+
+  cy.wait("@updateConnection").then((interception) => {
+    assert.isNotNull(interception.response?.statusCode, "200");
+  });
+
+  checkSuccessResult();
+};
+
 export const checkSuccessResult = () => {
   cy.get(successResult).should("exist");
 };
 
-export const confirmStreamConfigurationChangedPopup = () => {
+export const confirmStreamConfigurationChangedPopup = ({ reset = false } = {}) => {
+  if (!reset) {
+    cy.get(resetModalResetCheckbox).click({ force: true });
+  }
   cy.get(saveStreamChangesButton).click();
 };
 
 export const toggleStreamEnabledState = (streamName: string) => {
   cy.get(streamSyncEnabledSwitch(streamName)).check({ force: true });
 };
+
+export const checkSchemaChangesDetected = ({ breaking }: { breaking: boolean }) => {
+  cy.get(schemaChangesDetectedBanner).should("exist");
+  cy.get(schemaChangesDetectedBanner)
+    .invoke("attr", "class")
+    .should("match", breaking ? /\_breaking/ : /nonBreaking/);
+  cy.get(schemaChangesBackdrop).should("exist");
+};
+
+export const checkSchemaChangesDetectedCleared = () => {
+  cy.get(schemaChangesDetectedBanner).should("not.exist");
+  cy.get(schemaChangesBackdrop).should("not.exist");
+};
+
+export const clickSchemaChangesReviewButton = () => {
+  cy.get(schemaChangesReviewButton).click();
+  // cy.get(schemaChangesReviewButton).should("be.disabled");
```

Comment:
Why commented?
We need to enable the new table?

Reply:
This was related to not disabling the button in the loading state, which is fixed in #20737